code/game/objects/items/toys/mech_toys.dm

```diff
@@ -0,0 +1,642 @@
+/*
+ * Mech toys (previously labeled prizes, but that's unintuitive)
+ * Mech toy combat
+ */
+ 
+// Mech battle special attack types.
+#define SPECIAL_ATTACK_HEAL 1
+#define SPECIAL_ATTACK_DAMAGE 2
+#define SPECIAL_ATTACK_UTILITY 3
+#define SPECIAL_ATTACK_OTHER 4
+
+// Max length of a mech battle
+#define MAX_BATTLE_LENGTH 50
+
+/obj/item/toy/mech
```

Comment:
My only gripe is that its /mech and not /mecha to match how actual mechas are pathed in code

Reply:
I'd honestly just rename 'mecha' to 'mech'. 'mecha' sounds absurdly silly to me. :p

Reply:
The description of the toys is `Mini-Mecha action figure!`. They're mechas.